Nodejs/Product/Nodejs/TypingsAcquisition.cs

```diff
@@ -28,22 +28,47 @@
 
 namespace Microsoft.NodejsTools {
     internal class TypingsAcquisition {
-        private const string Tsd = "tsd";
-        private const string TsdExe = Tsd + ".cmd";
+        private const string TypingsAcquisitionTool = "typings";
```

Comment:
We may just want to rename these `Tool` or something similar since `TypingsAcquisition` is already in the class name.